acceptancetests/assess_cloud_display.py

```diff
@@ -37,7 +37,7 @@ def remove_display_attributes(cloud):
     # bug #1645783.
     defined = cloud.pop('defined')
     assert_equal(defined, 'local')
-    description = cloud.pop('description', None)
+    description = cloud.pop('description', "")
```

Comment:
`# Pop Nones, which are "errors from parsing the yaml"`

From the comment below, i think the change is wrong OR we change the comment.

Reply:
The comment is related to the code below not above. We pop them because somehow the code accesses them to check them later. 
As they are using direct access this will eventually lead to an `exception` thus we pop them below.
But pop is the wrong wording. It should be delete :\